manifests/03-rbac-logging.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: logging-configmap-reader
```

Comment:
@jcantrill just to verify, this role doesn't already exist in some format that we should bind to instead.

Reply:
@benjaminapetersen This is so the console can read our configmap that has route info in it?  Maybe this? https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/visualization.go#L405-L443 Something  we should consider removing?

Reply:
We will need to add the Role and RoleBinding to the cluster-logging-operator manifest, due to CI error that occurs when creating the `logging-configmap-reader` Role, since the `openshift-logging` namespace is not present on the cluster by default

Reply:
Ah, yes so:

```yaml
Role: sharing-config-reader
RoleBinding: openshift-logging-sharing-config-reader-binding (system:authenticated)
```
That was basically my question, should we use this for `console-operator` instead of creating new Role & RoleBinding for ourselves.